src/api/app/models/relationship.rb

```diff
@@ -127,7 +127,7 @@ def self.with_groups_and_roles
     with_groups_and_roles_query.pluck('groups.title', 'roles.title')
   end
 
-  def create_event_before_delete
+  def create_relationship_deletion_event
```

Comment:
IMO `create_relationship_delete_event` is better

Reply:
done